packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -362,7 +362,8 @@ class BouncingScrollPhysics extends ScrollPhysics {
 ///    behavior.
 ///  * [GlowingOverscrollIndicator], which is used by [ScrollConfiguration] to
 ///    provide the glowing effect that is usually found with this clamping effect
-///    on Android.
+///    on Android. By default, [GlowingOverscrollIndicator]'s glow adopts the
+///    [ThemeData.accentColor] when using a [MaterialApp].
```

Comment:
I sort of read that as suggesting that the logic is in GlowingOverscrollIndicator, rather than in MaterialApp, but I guess that's ok.

Reply:
flipped the sentence around